static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -34,12 +35,31 @@ define([
                  $,
                  defaults,
                  find,
-                 svgs) {
+                 svgs,
+                 fetch) {
 
         // change messageCode to force redisplay of the message to users who already closed it.
         // messageCode is also consumed by .../test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js
         var messageCode = 'engagement-banner-2017-01-11';
 
+        var SECONDARY_BUTTON_SELECTOR = '.secondary';
+
+        //Remind me form selectors
+        var REMIND_ME_FORM_SELECTOR = '.membership__remind-me-form';
```

Comment:
Maybe just `remind_me_form`? There is suddenly a lot of CAPS everywhere, which makes this harder for me to parse. YMMV

Since these are all selectors, maybe not necessary to append `_SELECTOR` to the variable name